lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -34,7 +35,7 @@ class UnusedBytes extends Audit {
    * @return {string}
    */
   static bytesToKbString(bytes) {
-    return Math.round(bytes / KB_IN_BYTES).toLocaleString() + ' KB';
+    return Util.formatBytesToKB(bytes);
```

Comment:
most of these audits should be ignoring savings less than 2-4 KB have you seen some that don't? we should fix those separately :)